addOns/spider/src/main/javahelp/org/zaproxy/addon/spider/resources/help/contents/automation.html

```diff
@@ -43,7 +43,7 @@ <H2>YAML</H2>
       postForm:                        # Bool: Whether the spider will submit POST forms, default: true
       processForm:                     # Bool: Whether the spider will process forms, default: true
       sendRefererHeader:               # Bool: Whether the spider will send the referer header, default: true
-      threadCount:                     # Int: The number of spider threads, default: 2
+      threadCount:                     # Int: The number of spider threads, default: 2 * Number of available processor cores
```

Comment:
This is not correct, either revert or change the value in the job parameters (I think the latter is better for consistency).

Reply:
Hey, there is a few places where Constants.getDefaultThreadCount() gets used within the spiderJob

![image](https://github.com/user-attachments/assets/b1e142e0-30da-4d98-9c4d-ebce71129c11)
![image](https://github.com/user-attachments/assets/87bb1b0b-0950-4f7f-9360-695040d671e9)
![image](https://github.com/user-attachments/assets/00ff1400-ca14-4a48-8514-f1defb61c5f4)

Some of the tests also intercept getDefaultThreadCount() which I take as further evidence that this is used to set the default:
![image](https://github.com/user-attachments/assets/109b232d-2757-4b28-b5e0-b4e966c10046)

Everywhere that I look, that method seems to be used, could you please help me understand why

Reply:
I think I did find what you are referring to:

![image](https://github.com/user-attachments/assets/275b9eb2-7361-4ffc-9cb4-eedf8060de0a)